iOS .ipa files - PPQS error (#1770)
saucelabs/sauce-docs: docs/mobile-apps/automated-testing/ipa-files.md

- `com.apple.developer.associated-domains`
- `com.apple.security.application-groups`
- `keychain-access-groups`

## Common Errors

> #### Unable to Verify App

Reviewer: ```suggestion
### Unable to Verify App
```
H4 heading after an H2 heading.

> If you are facing the issue where the app crashes with a red screen and an "Unable to Verify App" popup:
> <img src={useBaseUrl('img/mobile-apps/verify-app-error.png')} alt="Mobile app settings navigation" width="350"/>
> 
>  that means your proxy might be blocking Apple's signature check for installing custom enterprise apps on iOS. Apple has recently started rolling out a new signature verification and PPQS check for new prov. profiles and our new accounts. During installation, Apple sends an initial API request to verify the signature of the app.

Reviewer: Suggestion to reorg this content to explain the cause and error first, then provide the workaround.

```suggestion
Apple has a signature verification and PPQS check for new provisioning profiles and new accounts. During installation, Apple sends an API request to verify the signature of the app.

If the app crashes with a red screen and an "Unable to Verify App" error appears (as shown below), your proxy might be blocking Apple's signature check for installing custom enterprise apps on iOS.

<img src={useBaseUrl('img/mobile-apps/verify-app-error.png')} alt="Mobile app settings navigation" width="350"/>
```

We recommend that you try the following workaround:

1. Disable the proxy for the device you are using to install the app.

> 2. If the above solution does not work, please try using a different network without the proxy.

Reviewer: ```suggestion
2. If the above solution does not work, try using a different network without the proxy.
```
